### DISCUSSION TITLE
Alternative Discord verification

### DISCUSSION THREAD

**Discussion by phated** [grain-lang/grain#1370] [Announcements]
In July 2022, the Grain discord moderation team decided to enable Discord's "Phone Number Verification" to protect our users from spam bots and scams. While we realize this is heavy-handed, this is one of the few tools Discord provides to protect our members.

If you are unable to verify your phone number for some reason, please add a message here with your Discord handle and we can manually approve you.

By posting in this thread, you also agree to adhere to our Code of Conduct. You can find a copy of it here: https://github.com/grain-lang/grain/blob/main/CODE_OF_CONDUCT.md

**Comment by phated** [comment]
cc @kee-oth

**Comment by kee-oth** [comment]
My Discord handle is keeoth#7621

Thanks, @phated!

**Reply by phated** [reply]
Approved. Welcome!

**Comment by ghost** [comment]
My Disord handle is [Phosra#8780](https://discord.com/users/592759299248488468); I was looking to join your community, but uncomfortable with linking a number to Discord, so I was directed here by @spotandjake.

**Comment by jtenner** [comment]
Hey! Looks like I can't verify my phone number. I am definitely a wasm aficionado who worked on the AssemblyScript compiler/testing framework as-pect. 

My github is casual_bundows_enjoyer.

**Comment by silent-and-deadly** [comment]
I am a C programmer that wants to develop for the WASM/WASI targets but without the confusion of Emscripten.
my discord handle is meta.syntactic

**Reply by ospencer** [reply]
You're all set!

**Comment by ghost** [comment]
interested in trying grain
discord handle is
```
white__moose
```
with **two** underscores between white and moose `white_ _moose`

**Reply by ospencer** [reply]
All set!
